src/engine/shared/config.cpp

```diff
@@ -4,6 +4,7 @@
 #include <engine/console.h>
 #include <engine/storage.h>
 #include <engine/shared/config.h>
+#include <game/version.h>
```

Comment:
Seems like this causes cmake-based builds to fail... it seems like it is not viable to define `SETTINGS_FILENAME` in version.h?

Reply:
Probably have to change something in the build order, so it creates `generated/nethash.cpp` when compiling this.